audio/orca/Portfile

```diff
@@ -63,3 +63,5 @@ notes "
 To get started, see ${prefix}/share/doc/${name}/README.md and
 ${prefix}/share/examples/${name}/.
 "
+
+livecheck.type          none
```

Comment:
why disable this? It probably doesn't give the "usual" message as the use of a hash instead of a tag isn't ideal.. But at least it will tell you if there have been new commits, disabling the livcheck will not give you anything at all.

Reply:
This is just a cosmetic fix for [portfile page](https://ports.macports.org/port/orca/details/). The app has no versions and is only updated periodically, so why not add this.

Reply:
We don't have a way of handling livechecks in such a circumstance?